lib/origin_simulator/simulation.ex

```diff
@@ -115,10 +104,35 @@ defmodule OriginSimulator.Simulation do
     {:noreply, Map.put(state, route, %{state[route] | status: status, latency: latency})}
   end
 
+  @impl true
+  def handle_continue(:setup_default_recipe, state) do
+    recipe = File.read!(File.cwd!() <> "/examples/default.json") |> OriginSimulator.Recipe.parse()
+    simulation = ingest_recipe(state, recipe)
+
+    {:noreply, simulation}
+  end
+
+  defp ingest_recipe(state, new_recipe) do
+    :ok = Payload.fetch(:payload, new_recipe)
+
+    route = new_recipe.route
+    simulation = get(state[route])
+
+    Enum.map(new_recipe.stages, fn item ->
+      Process.send_after(
+        self(),
+        {:update, route, item["status"], Duration.parse(item["latency"])},
+        Duration.parse(item["at"])
+      )
+    end)
+
+    Map.put(state, route, %{simulation | recipe: new_recipe})
+  end
+
   defp get(nil), do: default_simulation()
   defp get(current_state), do: current_state
 
-  defp default_simulation(), do: %{recipe: nil, status: 406, latency: 0}
+  defp default_simulation, do: %{recipe: nil, status: 406, latency: 0}
```

Comment:
do we need this function now?

Reply:
yes... Simulation.ex:119

That's what I want to improve ...  next :)